CONTRIBUTING.md

```diff
@@ -125,3 +125,5 @@ Triton requires that all contributors (or their corporate entity) send
 a signed copy of the [Contributor License
 Agreement](https://github.com/NVIDIA/triton-inference-server/blob/master/Triton-CCLA-v1.pdf)
 to triton-cla@nvidia.com.
+*NOTE*: Contributors with no company affiliation can fill their name on
+the form. For example, `John Doe (Self)`.
```

Comment:
"Additionally, you can use N/A in the 'Corporation address' and 'Corporation name' fields when filling the form as an individual contributor."

Reply:
I think that is fine, yes